content/api/rest-sdk.textile

```diff
@@ -0,0 +1,242 @@
+---
+title: Constructor
+section: api
+index: 10
+languages:
+  - javascript
+  - nodejs
+  - php
+  - python
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - go
+  - flutter
+jump_to:
+  Constructor:
```

Comment:
I've called this `Constructor object` if we want to standardise - not fussed which one we choose.

Reply:
I just used Constructor because that was the section title. However, Constructor object is more explicit, so I've changed to match your approach.